include/projfs_vfsapi.h

```diff
@@ -167,13 +167,15 @@ PrjFS_Result PrjFS_WritePlaceholderFile(
 PrjFS_Result PrjFS_ConvertDirectoryToPlaceholder(
     _In_    const char*                             relativePath
 );
-
+#endif
 
 PrjFS_Result PrjFS_WriteSymLink(
+    _In_    const PrjFS_MountHandle*                mountHandle,
```

Comment:
Looks like you've already got this heading upstream in github/VFSForGit@ed0badfbf9d08abfeb9d985610bc8918b1868a39 -- nice!!